css/80_app.css

```diff
@@ -1706,9 +1706,12 @@ a.hide-toggle {
 .form-field-input-text > input:only-of-type,
 .form-field-input-tel > input:only-of-type,
 .form-field-input-email > input:only-of-type,
-.form-field-input-url > input:only-of-type {
+.form-field-input-url > input:only-child {
```

Comment:
I can't hit this case. Can you please just quickly post why did you add this .css change?

Reply:
the DOM looks like this:
```html
<div class="form-field-input-url">
	<input type="text" />
</div>

<!-- or -->

<div class="form-field-input-url">
	<input type="text" />
    <button></button>
</div>
```

`input:only-of-type` won't work to identify the second case, because the button is a different element. I think the rule was written assuming that `<input type=button />` would be used instead of `<button> </button>`. 
The existing code (line 106 & 131 in that line) uses `<button>` so I wanted to be consistent with that. I can change it to `<input type=button />` and remove the css change if you prefer.

Reply:
Got it, the current version is ok, and thanks for paying attention to the details such as eliminating border radius for the corner next to the button.